.circleci/config.yml

```diff
@@ -61,7 +61,7 @@ workflows:
       - lint
       - test
       - build-jsonnet
-      - container-push:
+      - container-nightly-push:
```

Comment:
hm - it's not nightly, it's on master push, no?

Reply:
Yeah, but I couldn't find a better name. Called it "trunk" build reminds of me SVN too much 😂 

Any suggestion on a better name is welcome.